bundles/org.openhab.binding.telegram/README.md

```diff
@@ -155,12 +155,19 @@ These actions will send a message to all chat ids configured for this bot.
 
 ### Actions to send messages to a particular chat
 
-Just put the chat id (must be a long value!) as the first argument to one of the above mentioned APIs:
+Just put the chat id (must be a long value!) followed by an "L" as the first argument to one of the above mentioned APIs:
 
 ```
 telegramAction.sendTelegram(1234567L, "Hello world!")
 ```
 
+You can also define a global variable to use in all your rules file:
+
+```
+val Id = 1234567L
+telegramAction.sendTelegram(Id, "Hello world!")
+```
+
```

Comment:
Hm... Here I tend to say that defining a global variable instead of passing the actual value is something that is not related to the Telegram binding and applies to all other bindings that define an action where you can pass something. If somebody doesn't know about that, I would refer to the general rule guide of OH (https://www.openhab.org/docs/configuration/rules-dsl.html).

Reply:
Agree, removed